### PR TITLE
swaps: output disabled explainers

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes TEAM2-513
Figma link (if any):

## What changed (plus any additional context for devs)

- Pressing output input will prompt a new explainer when doing a cross chain swap or bridge
- Confirm swap button will say `Tap / Hold to Bridge`
- keyboard won't hide while switching network  https://www.loom.com/share/8f4b4d2d84734c82867ee4b58fbc69c0

## Screen recordings / screenshots

https://www.loom.com/share/b7e38fa4799641cdb08ec4f2d7ad44d0

## What to test

follow the video and check explainers make sense

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/screens/ExchangeModal.tsx [hunk 1]
```diff
@@ -267,15 +268,22 @@ export default function ExchangeModal({
     const currentNetwork = ethereumUtils.getNetworkFromChainId(chainId);
     const isCrosschainSwap =
       crosschainSwapsEnabled && inputNetwork !== outputNetwork;
-
+    const isBridgeSwap = inputCurrency?.symbol === outputCurrency?.symbol;
```
Comment (skylarbarrera) [inline src/screens/ExchangeModal.tsx:271]: I feel like this could be just isBridge

**Review by skylarbarrera — APPROVED** [review]
👌

File: src/screens/ExplainSheet.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ import {
 } from '@/utils';
```
Comment (jkadamczyk) [inline src/screens/ExplainSheet.js:36]: This whole file is WAY TOO LONG
We should try splitting it into smaller pieces

**Review by jkadamczyk — APPROVED** [review]
I approve your changes Esteban, but the way ExplainSheet.js file looks like is straight out bad, it grew to be too long and I think we should definitely split it into smaller pieces

File: src/screens/ExplainSheet.js (hunk 2 above)
Comment (estebanmino) [reply src/screens/ExplainSheet.js:36]: yep

**Pull request merged**